backend/btrixcloud/models.py

```diff
@@ -1940,6 +1948,9 @@ class Subscription(BaseModel):
     planId: str
 
     futureCancelDate: Optional[datetime] = None
+    # pylint: disable=C0301
+    "When in a trial, future cancel date is the trial end date; when not in a trial, future cancel date is the date the subscription will be canceled, if set"
+
```

Comment:
```suggestion
    # When in a trial, future cancel date is the trial end date.
    # When not in a trial, future cancel date is the date the subscription will
    # be cancelled, if set.
    futureCancelDate: Optional[datetime] = None

```

Think a regular comment here might be more appropriate. Unless there's something I'm missing?

Reply:
At least in VSCode and other editors that use the Python LSP server, this shows up as text wherever this property is used: 

<img width="739" height="191" alt="Image" src="https://github.com/user-attachments/assets/af254c4a-bb0d-45cf-9c28-a34cdf544a9a" />

I figured it'd make sense to do this rather than a normal comment because we use this in a way that's not necessarily intuitive from just the property name — this way the additional information shows up wherever it's used.

Reply:
Oh cool! Based on this pydantic issue, it seems maybe it should be wrapped in triple quotes as a docstring?: https://github.com/pydantic/pydantic/issues/5656

But admittedly this is new to me so I may well be wrong!

Reply:
Oh nice, yeah — just requires `model_config = ConfigDict(use_attribute_docstrings=True)` to be set for that to work.